precompiles/Modexp.yul

```diff
@@ -28,6 +28,86 @@ object "ModExp" {
             }
 
+            function getLimbValueAtOffset(limbPointer, anOffset) -> limbValue {
+                limbValue := mload(add(anOffset, limbPointer))
+            }
+
+            function storeLimbValueAtOffset(limbPointer, anOffset, aValue) {
+                    mstore(add(limbPointer, anOffset), aValue)
+            }
+
+            function subLimbsWithBorrow(leftLimb, rightLimb, limbBorrow) -> substractionResult, returnBorrow {
+                let rightPlusBorrow := add(rightLimb, limbBorrow)
+                substractionResult := sub(leftLimb, rightPlusBorrow)
+                if gt(substractionResult, leftLimb) {
+                 returnBorrow := 1
+                }
+            }
+
+            function bigUintSubstractionWithBorrow(lhsPointer, rhsPointer, numberOfLimbs, resultPointer) -> resultPointer, borrow {
+                let leftIthLimbValue
+                let rightIthLimbValue
+                let ithLimbBorrowResult 
+                let ithLimbSubstractionResult 
+                let borrow := 0
+                let limbOffset := 0
+                for {let i := numberOfLimbs} gt(i, 0) {i := sub(i, 1)} {
+                    limbOffset := mul(sub(i,1), 32)
+                    leftIthLimbValue := getLimbValueAtOffset(lhsPointer, limbOffset)
+                    rightIthLimbValue := getLimbValueAtOffset(rhsPointer, limbOffset)
+                    ithLimbSubstractionResult, borrow :=
+                                               subLimbsWithBorrow(leftIthLimbValue, rightIthLimbValue, borrow)
+                    storeLimbValueAtOffset(resultPointer, limbOffset, ithLimbSubstractionResult)
+
+                }
+            }
+
+
+
+
+            // Test4: Big number 4 limb substraction
+            // First Number:
+            //  - First Limb: 0xFFFF
+            //  - Second Limb: 0xAAAA
+            //  - Third Limb: 0xFFFF
+            //  - Fourth Limb: 0xAAAA
+            // Second Number:
+            //  - First Limb: 0xAAAA
+            //  - Second Limb: 0xFFFF
+            //  - Third Limb: 0xAAAA
+            //  - Fourth Limb: 0xFFF
+            //  
+            // Should print:
+            // 0x5554 (first limb)
+            // 0xffffffffffffaaab (second limb)
+            // 0x5555 (third limb)
+            // 0x9aab (fourth limb)
+            
+
+            mstore(0x00, 0xffff)
+            mstore(0x20, 0xaaaa)
+            mstore(0x40, 0xffff)
+            mstore(0x60, 0xaaaa)
+
+            mstore(0x80, 0xaaaa)
+            mstore(0xa0, 0xffff)
+            mstore(0xc0, 0xaaaa)
+            mstore(0xe0, 0xfff)
+
+
+            let retStart := 0x100
+            let _, borrow := bigUintSubstractionWithBorrow(0x00, 0x80, 4, retStart)
+
+            let res := mload(retStart)
+            let res2 := mload(add(retStart, 0x20))
+            let res3 := mload(add(retStart, 0x40))
+            let res4 := mload(add(retStart, 0x60))
+
+            console_log(res)
```

Comment:
Calls to console_log should be removed

Reply:
I fixed this @ [3643287](https://github.com/lambdaclass/zksync_era_precompiles/pull/149/commits/3643287ecfa28e4e12a898578cf4d48542e7dc6e)